js/reveal.js

```diff
@@ -368,7 +370,7 @@ export default function( revealElement, options ) {
 		// Limit the size of certain elements to the dimensions of the slide
 		createStyleSheet( '.reveal section>img, .reveal section>video, .reveal section>iframe{max-width: '+ slideWidth +'px; max-height:'+ slideHeight +'px}' );
 
-		document.body.classList.add( 'print-pdf' );
+		document.querySelector('html').classList.add( 'print-pdf' );
```

Comment:
This should use `document.documentElement` instead of `querySelector('html')`, to match how the rest of reveal.js accesses it.

Reply:
Will do